gprofiler/merge.py

```diff
@@ -73,7 +76,8 @@ def parse_perf_script(script: str):
             logger.exception(f"Error processing sample: {sample}")
 
 
-def merge_perfs(perf_all: Iterable[Mapping[str, str]], process_perfs: Mapping[int, Mapping[str, int]]) -> str:
+def merge_perfs(perf_all: Iterable[Mapping[str, str]], process_perfs: Mapping[int, Mapping[str, int]],
```

Comment:
the format of the output file has changed (and now each sample includes the container-name);
@Jongy - should we strip the container names from the file when generating an svg file on the remote?
(please bare in mind that the logic for extracting a meaningful name from the k8s-pods's containers is not present here and only exists in the backend)

Reply:
In the remote - we need to discuss.

Locally - I think it's nice to keep, but we need @alonbentamar to show an example of how it looks (it might be too long and be rendered badly; in that case we should remove)

Reply:
this PR should not get in till we have fully supported in the backend as well @alonbentamar @ezean21 

regarding the remote - lets talk :)

Reply:
> In the remote - we need to discuss.
> 
> Locally - I think it's nice to keep, but we need @alonbentamar to show an example of how it looks (it might be too long and be rendered badly; in that case we should remove)

It will be too long... for sure. Just think about a k8s container...
I think I'd prefer stripping it down.

Reply:
k can strip it